openwisp_notifications/templates/openwisp_notifications/emails/unsubscribe_footer.html

```diff
@@ -0,0 +1,2 @@
+{% load i18n %}
+<p style="margin-bottom: 0px; font-size: small">{% blocktrans %}To unsubscribe from these notifications, <a href="{{ unsubscribe_url }}">click here</a>.{% endblocktrans %}</p>
```

Comment:
What do you think of centering this?

[Why You Should Never Write “Click Here”](https://careerfoundry.com/en/blog/ux-design/click-here/) applies here as well.